patrol/cardinality.go

```diff
@@ -62,18 +62,40 @@ func (p *Patrol) getTopCardinalities() error {
 	}
 
 	for _, s := range highCardSeries {
-		mrc := config.GenerateMetricRelabelConfig(s)
-		prom.ReUnmarshal(&mrc)
+		mrc, err := config.GenerateMetricRelabelConfig(s)
+		if err != nil {
```

Comment:
Is there any point in continuing here if this fails? Or should you move to the next highCardSeries?

Reply:
Yup, seems I missed a `continue`.

Reply:
I kinda feel like I should break up this function, it's too meaty. Another PR, perhaps.